charts/quickwit/values.yaml

```diff
@@ -492,4 +568,4 @@ resources: {}
   #   memory: 128Mi
   # requests:
   #   cpu: 100m
-  #   memory: 128Mi
+#   memory: 128Mi
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed